flowgate-api/src/main/java/com/vmware/flowgate/service/AssetService.java

```diff
@@ -59,6 +61,29 @@
    private static final int SERVER_ASSET_NAME_TIME_OUT = 7200;
    private static final int LIMIT_RESULT = 100;
    private static final String UnknownOulet = "unknownoutlet";
+   private static final int timestampArrayLength = 2;//The extraInfo of peakPower and minimumPower metric include since time and event time.
+   private static Map<String, String> databaseUnitAndOutputUnitMap = null;
+   static {
+      databaseUnitAndOutputUnitMap = new HashMap<String, String>();
+      databaseUnitAndOutputUnitMap.put(RealtimeDataUnit.Amps.toString(), MetricUnit.A.toString());
+      databaseUnitAndOutputUnitMap.put(RealtimeDataUnit.Volts.toString(), MetricUnit.V.toString());
+      databaseUnitAndOutputUnitMap.put(RealtimeDataUnit.Celsius.toString(), MetricUnit.C.toString());
+      databaseUnitAndOutputUnitMap.put(RealtimeDataUnit.Fahrenheit.toString(), MetricUnit.F.toString());
+      databaseUnitAndOutputUnitMap.put(RealtimeDataUnit.KW.toString(), MetricUnit.KW.toString());
+      databaseUnitAndOutputUnitMap.put(RealtimeDataUnit.Percent.toString(), MetricUnit.PERCENT.toString());
+      databaseUnitAndOutputUnitMap.put(RealtimeDataUnit.KBps.toString(), MetricUnit.KBps.toString());
+      databaseUnitAndOutputUnitMap.put(RealtimeDataUnit.Mhz.toString(), MetricUnit.Mhz.toString());
+      databaseUnitAndOutputUnitMap.put(RealtimeDataUnit.KB.toString(), MetricUnit.KB.toString());
+      databaseUnitAndOutputUnitMap.put(MetricUnit.A.toString(), MetricUnit.A.toString());
+      databaseUnitAndOutputUnitMap.put(MetricUnit.AMPS.toString(), MetricUnit.A.toString());
```

Comment:
double check if some place use amps, change to A

Reply:
Done